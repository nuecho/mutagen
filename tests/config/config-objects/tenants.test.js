cfgObjectTests("tenants", "config-objects/tenants-config.json", null, 1);
