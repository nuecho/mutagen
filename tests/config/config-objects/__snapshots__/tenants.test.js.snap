// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[mutagen tenants config import-export] should create the tenants: the tenants have been created 1`] = `
Object {
  "stderr": "",
  "stdout": "+ tenant [cfgObjectTest]
Completed. 1 object(s) imported.
",
}
`;

exports[`[mutagen tenants config import-export] should update the modified tenants: the modified tenants have been updated 1`] = `
Object {
  "stderr": "",
  "stdout": "~ tenant [cfgObjectTest]
Completed. 1 object(s) imported.
",
}
`;

exports[`[mutagen tenants config import-export] should update the same tenants: the tenants have been updated 1`] = `
Object {
  "stderr": "",
  "stdout": "~ tenant [cfgObjectTest]
Completed. 1 object(s) imported.
",
}
`;
